src/pages/Sell/ConfirmationPage.jsx

```diff
@@ -1,9 +1,72 @@
 import { useSellContext } from '../../contexts/SellContext';
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { Container, Button, Card, Row, Col } from 'react-bootstrap';
 import './ConfirmationPage.css'
+import { getExchangeLocationAndSchedule } from '../../api/organization';
+import { useAuthContext } from '../../contexts/AuthContext';
+
+function getDayName(index) {
+  const days = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'];
+  return days[index];
+}
 
 function NextSteps() {
+  const [organizationData, setOrganizationData] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+  const { currentUser } = useAuthContext();
+
+  useEffect(() => {
+
+    async function fetchData() {
+      if (!currentUser || !currentUser.organization_id) {
+        setError('User or organization ID is not available');
+        setLoading(false);
+        return;
+      }
+      
+      try {
+        const data = await getExchangeLocationAndSchedule(currentUser.organization_id);
+        setOrganizationData(data);
+      } catch (err) {
+        setError(err.message);
+      } finally {
+        setLoading(false);
+      }
+    }
+    fetchData();
+  }, [currentUser]);
+
+  // Create a complete schedule with all days, including those without times
+  const completeSchedule = [
+    { day: 'Sun', start: '', end: '' },
+    { day: 'Mon', start: '', end: '' },
+    { day: 'Tue', start: '', end: '' },
+    { day: 'Wed', start: '', end: '' },
+    { day: 'Thu', start: '', end: '' },
+    { day: 'Fri', start: '', end: '' },
+    { day: 'Sat', start: '', end: '' }
+  ];
+
+  // Fill in the schedule with data from organizationData
+  if (organizationData && organizationData.schedule) {
+    organizationData.schedule.forEach((day, index) => {
+      if (day && day.start && day.end) {
+        completeSchedule[index] = {
+          day: getDayName(index),
+          start: day.start,
+          end: day.end
+        };
+      }
+    });
+  }
+
+  // Filter out days with missing times and format the schedule string
+  const scheduleString = completeSchedule
+    .filter(day => day.start && day.end) // Exclude days with missing times
+    .map(day => `${day.day}: ${day.start} - ${day.end}`)
+    .join('; ');
```

Comment:
Since the same function is used in the confirmation page and the buysuccess page it would be good to eventually just put it in one file but its fine for now